api/v1/db/oracledb/player-dao.js

```diff
@@ -65,12 +66,57 @@ const getPlayerByGameIdAndPlayerId = async (id, pid) => {
     if (_.keys(groupedRawPlayers).length > 1) {
       throw new Error('Expect a single object but got multiple results.');
     } else {
-      const serializedPlayer = serializePlayer(rawPlayers, id);
+      const serializedPlayer = serializePlayer(rawPlayers, id, isPost);
       return serializedPlayer;
     }
   } finally {
     connection.close();
   }
 };
 
-module.exports = { getPlayersByGameId, getPlayerByGameIdAndPlayerId };
+const postPlayerByGameId = async (body, gameId) => {
+  const connection = await conn.getConnection();
+  try {
+    body = body.data.attributes;
+    body.outId = {
+      type: oracledb.NUMBER,
+      dir: oracledb.BIND_OUT,
+    };
+
+    const { playerCards } = body;
+    delete body.playerCards;
+    const postSqlQuery = `
+    INSERT INTO PLAYERS (MEMBER_ID, GAME_ID, PLAYER_BET, STATUS_ID)
+    VALUES (:memberId,
+           ${gameId},
+           :playerBet,
+           (SELECT STATUS_ID FROM STATUSES WHERE STATUSES.STATUS = :playerStatus)) RETURNING PLAYER_ID INTO :outId
+    `;
+    const rawPlayer = await connection.execute(postSqlQuery, body, { autoCommit: true });
+    const promiseArray = [];
+    const playerId = rawPlayer.outBinds.outId[0];
+    console.log(playerId);
+    console.log(playerCards);
+    _.forEach(playerCards, (card) => {
+      console.log(card);
+      const cardSqlQuery = `INSERT INTO PLAYER_CARDS (PLAYER_ID, CARD_ID) VALUES
+      (${playerId},
+       (SELECT CARD_ID FROM CARDS C
+        INNER JOIN CARD_SUITS CS ON CS.SUIT_ID = C.CARD_SUIT_ID
+        INNER JOIN CARD_NUMBERS CN ON CN.CARD_NUMBER_ID = C.CARD_NUMBER_ID
+        WHERE SUIT = :cardSuit AND CARD_NUMBER = :cardNumber
+       )
+      )`;
+      promiseArray.push(connection.execute(cardSqlQuery, card, { autoCommit: true }));
+    });
```

Comment:
Generally, when you see you're following the pattern
```
_.forEach(array, (item) => {
  newItem = doSomething(item)
  newArray.push(transformedItem)
});
```
You should replace this with `_.map()` instead